client/src/components/dashboardPage/Dashboard.js

```diff
@@ -27,54 +29,229 @@ const styles = theme => ({
   useEffect(() => {
-    // props.getTeachers(options);
-    // props.getStudents(options);
     props.getClasses();
-    // props.getQuestions(options);
   }, []);
-  const { teachers, students, allClasses, questions, classes } = props;
-  //   console.log('Teachers Check:', teachers)
-  //   console.log('Students Check:', students)
-  //   console.log('Classes Check:', classes)
-  //   console.log('Questions Check:', questions)
+  const { allClasses, classes } = props;
+  const testClasses = [
+    {
+      classname: 'FSW438',
+      numOfStudents: 84,
+      participationRate: 76,
+      refreshrsEmailed: 67,
+      class_id: 1
+    },
+    {
+      classname: 'Android 74',
+      numOfStudents: 84,
+      participationRate: 78,
+      refreshrsEmailed: 36,
+      class_id: 2
+    },
+    {
+      classname: 'Android 74',
+      numOfStudents: 84,
+      participationRate: 78,
+      refreshrsEmailed: 36,
+      class_id: 3
+    },
+    {
+      classname: 'Android 74',
+      numOfStudents: 84,
+      participationRate: 78,
+      refreshrsEmailed: 36,
+      class_id: 4
+    }
+  ];
+  const testRefreshrs = [
+    {
+      refreshrName: 'CSS Basics',
+      classesAssigned: 84,
+      refreshr_id: 1
+    },
+    {
+      refreshrName: 'Relational Databases',
+      classesAssigned: 21,
+      refreshr_id: 2
+    },
+    {
+      refreshrName: 'Relational Databases',
+      classesAssigned: 21,
+      refreshr_id: 3
+    },
+    {
+      refreshrName: 'Relational Databases',
+      classesAssigned: 21,
+      refreshr_id: 4
+    },
+    {
+      refreshrName: 'Relational Databases',
+      classesAssigned: 21,
+      refreshr_id: 5
+    }
+  ];
   return (
     <Grid className={classes.wrapper}>
-              <h1 style={{color:'white'}}>User: {name}</h1>
-
-      {/* <div className={classes.containers}>
-        <h5>Students</h5>
-        {students.map(student => (
-          <p className={classes.lists}>{student.email}</p>
-        ))}{' '}
-      </div>
-
-      <div className={classes.containers}>
-        <h5>Teachers</h5>
-        {teachers.map(teacher => (
-          <p className={classes.lists}>{teacher.email}</p>
-        ))}
-      </div> */}
-      <div className={classes.containers}>
-        <h5>Classes</h5>
-        {allClasses.map(c => (
-          <p key={c.class_id} className={classes.lists}>{c.classname}</p>
+      {console.log('PROPS', allClasses)}
+      {/* cant figure out what to do w/ the username right now */}
+      {/* <Typography component="h2" color="secondary">
+        Welcome {name}, 
+      </Typography> */}
+      {/* end username */}
+      <Typography
+        component="h2"
+        color="secondary"
+        className={classes.cardSectionLabels}
+      >
+        Classes:
+      </Typography>
+      <Grid className={classes.classContainer}>
+        {testClasses.map(c => (
+          <Card key={c.class_id} className={classes.classCard}>
```

Comment:
Once we have this hooked up to the db, we could just wrap each of these cards in a `<Link to=/classes/edit/${c.class_id}>`, and do something similar with the refreshrs